submissions/hall-of-fame/hall-of-fame.md

```diff
@@ -0,0 +1,10 @@
+## Mod spotlight: [Hall of Fame: Menu Simulations and Screensaver](https://mods.factorio.com/mod/HallOfFame) <author>Conor_</author>
+
+Back in [FFF #362](https://factorio.com/blog/post/fff-362), *V453000* introduced us to the new main menu for Factorio; real-time simulations of the game running in the background, piggybacking of the technology for the new and shiny [Tips and Tricks GUI](https://www.factorio.com/blog/post/fff-361). This was a great new feature, allowing for a much more interesting and spidertron-filled welcoming to the game.
+
+![New Factorio menu](media/fff-362-menu-simulation.mp4)
+*Source: [FFF #362](https://factorio.com/blog/post/fff-362)*
+
+Of course, Wube being the ~~godlike~~ fantastic humans that they are made it easy for anyone to create a mod adding their own main menu simulations. Our very own *stringweasel*, editor to the stars :p, took this as a challenge to continue his ongoing project, diving into the depths of Factorio of old (see Alt-F4's [#6](https://alt-f4.blog/ALTF4-6/#nauvis-archives-how-far-weve-come-stringweasel), [#9](https://alt-f4.blog/ALTF4-6/#nauvis-archives-how-far-weve-come-stringweasel), [#11](https://alt-f4.blog/ALTF4-11/#nauvis-archives-outdated-techniques-stringweasel), [#13](https://alt-f4.blog/ALTF4-13/#nauvis-archives-one-rocket-defence-per-minute-stringweasel) and [#21](https://alt-f4.blog/ALTF4-21/#nauvis-archives-from-mod-to-vanilla-stringweasel)). Out of this historical pedigree, mightily, like a majestic phoenix, as if it were handed down from the heavens, the [Factorio Hall of Hame mod](https://mods.factorio.com/mod/HallOfFame) rose to glory. *stringweasel* has painstakingly sought out interesting Factorio maps from the archives of Reddit, the forum and the deep bowels of the internet, providing over 30 different base snapshots for us to enjoy as we load into the game. From speedruns to high SPM monsters, spaghetti messes to beautifully laid out art pieces and even a [burner inserter masterpiece](https://alt-f4.blog/ALTF4-8/#i-love-factorio-because-of-burner-inserters-goose). I find myself loading up the game and pausing on the main menu for while, admiring a few interesting bases and getting inspired before I dive back into whatever spaghetti nightmare I left behind last session. Below is a small sneak peek into the awesomeness that is Hall of Fame but I recommend installing it for the full, arguably life-changing experience.
+
+![Video of Hall of Fame Simulations](media/hall-of-fame.mp4)
```

Comment:
Are those beehives as roboports? XD

I've spent a lot of time on a little trailer for the mod that's on YouTube. Would you mind if we link that instead? It will be HD and has a nice Wintergatan backtrack :P https://youtu.be/pBgLwkLgsWs

Reply:
> Are those beehives as roboports? XD

Not sure what you're talking about, roboports have always been bee themed :p

Linking to the trailer is a much better idea though so I'll change that over.